packages/@webex/widget-cc-station-login/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@webex/cc-station-login",
```

Comment:
Why are we removing the `widget` prefix here?

Reply:
Added widget prefix